stubs/sklearn/preprocessing/_data.pyi

```diff
@@ -135,11 +137,13 @@ class MaxAbsScaler(OneToOneFeatureMixin, TransformerMixin, BaseEstimator):
     def transform(self, X: spmatrix) -> spmatrix: ...
     @overload
     def transform(self, X: ArrayLike) -> ndarray: ...
+    @overload
```

Comment:
Are these extra `@overload` decorators correct? Aren't the decorated functions the implementations? Wouldn't there be an overlap between these and the pre-existing overloads?